src/pulse3D/excel_writer.py

```diff
@@ -435,7 +434,7 @@ def write_xlsx(
         max_y = int(max_force_of_recording)
     # waveform table
     continuous_waveforms = {
-        "Time (seconds)": pd.Series(interpolated_timepoints_secs[start_idx:end_idx] / MICRO_TO_BASE_CONVERSION)
+        "Time (seconds)": pd.Series(interpolated_timepoints_secs[:end_idx] / MICRO_TO_BASE_CONVERSION)
```

Comment:
have you tried this with an analysis that has windowed analysis params set?
just curious if it works because I had to add this for that.

Reply:
Yes I just tested it, and it still works with the changes. I think `interpolated_timepoints_secs` hold all data needed to graph taking into account the `start_time` param. So this should just collect all the points in the list up to end_idx. I'm not sure my explanation makes sense though?

Reply:
I only released the change this morning from `[0:end_index]` to `[start_idx, end_idx]`